R/shell_connection.R

```diff
@@ -20,7 +20,9 @@ shell_connection <- function(master,
                              remote,
                              extensions,
                              batch,
-                             scala_version = NULL) {
+                             scala_version = NULL,
+                             enable_spark_rapids_plugin = FALSE,
```

Comment:
I think we rather want `rapids` to be specified in `packages`, as in:

```r
sc <- spark_connect(master = "local", packges = "rapids")
```

And also to specify the configuration in `spark_config` with the `rapids.` prefix, as in:

```r
config <- spark_config()
config$rapids.some.key = "value"
config$rapids.some.other = "other"
```

And then extract the config options, remove the prefix and use as `spark_rapids_plugin_conf` internally.